hathor/indexes/manager.py

```diff
@@ -63,6 +64,22 @@ def enable_tokens_index(self) -> None:
         """Enable tokens index. It does nothing if it has already been enabled."""
         raise NotImplementedError
 
+    def _manually_initialize(self, tx_storage: 'TransactionStorage') -> None:
+        """ Initialize the tips indexes, populating them from a tx_storage that is otherwise complete.
+        """
+        # XXX: use tx_storage._topological_fast when it is available
```

Comment:
Should we create a `tx_storage.topological_sort()` that points to the desired method? The caller can either choose to use a specific method or the default one.

Reply:
I prefer not to. Both iterators still have very specific caveats, I think we still have to be careful with "recommending" an iterator. I think we could leave them internal for now.